### PR TITLE
Updated Issue template

### LINKED ISSUE #120: Improvement in Issue template
- Addition of checkbox in first question

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
**Status**:
- [x] Ready
- [ ] In Development
- [ ] On Hold

**Which issue does this PR fix?**: fixes #120 

Brief description of what this PR does.
This Issue template is now more understable and easy to fill.

### CODE REVIEW

**Comment by amarlearning** [comment]
I would say instead of showing the status of PR in a description, we should label for that.

Like if the 
1. PR is in development we can use <kbd>s.OnGoing</kbd> label for that.
2. PR is ready for review then we can use <kbd>s.ToReview</kbd> label for that.
3. PR is on hold then we can use <kbd>s.OnHold</kbd> label for that.

By the way, the reviewer can get to know the status of PR without even opening it.

**Comment by vaibhavsingh97** [comment]
@amarlearning yeah, we can remove but we should add that in description for others who are visiting our repo should get to know what these labels means. I would open an issue for further discussion

File: docs/ISSUE_TEMPLATE.md
Comment (nikhita) [inline docs/ISSUE_TEMPLATE.md]: if relevant, please include screenshots and logs.

File: docs/ISSUE_TEMPLATE.md
Comment (nikhita) [inline docs/ISSUE_TEMPLATE.md:7]: I like this. :+1:

**Review by nikhita — COMMENTED** [review]
LGTM except one minor change.

**Comment by nikhita** [comment]
> I would say instead of showing the status of PR in a description, we should label for that.

IIUC that should go in the PR template, correct? This is the Issue template. However, the subtleties aside, I agree on this point. I have noticed that many don't use the check boxes to update the status of their PR at all. Or even if they do, they don't update the status after it moves to the other stage.

Having labels would mean that only the maintainers would be able to add it (and honestly, I am fine with that). The reviewer for that PR can add the labels and can provide a more accurate "status".

> we should add that in description for others who are visiting our repo should get to know what these labels means.

Maybe a issue I faced personally but it took me an embarrassing amount of time to realize what `s` in `s.OnGoing` means. :stuck_out_tongue: Maybe we call it out explicitly? The labels can be `status: OnGoing`, `status: Needs Review`, `status: OnHold`? We can, of course, brainstorm if we need other labels too.

> I would open an issue for further discussion

Maybe we could get it done within this PR only? Update the Issue and PR templates at once? I have no strong preference on this. Feel free to do either way. :+1:

**Comment by vaibhavsingh97** [comment]
>IIUC that should go in the PR template, correct? This is the Issue template. However, the subtleties aside, I agree on this point. I have noticed that many don't use the check boxes to update the status of their PR at all. Or even if they do, they don't update the status after it moves to the other stage.

We needed bot who does this automatically, but this is least priority issue. Maintainers can do that manually 😄 

> Maybe a issue I faced personally but it took me an embarrassing amount of time to realize what s in s.OnGoing means. 😛 Maybe we call it out explicitly? The labels can be status: OnGoing, status: Needs Review, status: OnHold? We can, of course, brainstorm if we need other labels too.

I liked these lables over s.Review, s.ongoing, etc. We can add this but first we should add this requirement in an issue

File: docs/ISSUE_TEMPLATE.md
Comment (vaibhavsingh97) [reply docs/ISSUE_TEMPLATE.md]: I will change 👍

**Comment by nikhita** [comment]
Created https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/123.

I'll send a PR later today :+1:

**Comment by vaibhavsingh97** [comment]
@nikhita I can send PR as i am already working on this.

**Comment by nikhita** [comment]
> @nikhita I can send PR as i am already working on this.

Sounds good, thanks.

**Comment by vaibhavsingh97** [comment]
@nikhita Updated the PR. Please review 😄

File: docs/ISSUE_TEMPLATE.md
Comment (nikhita) [inline docs/ISSUE_TEMPLATE.md]: nit: **If** (with a capital I)

nit: please include ~a~ **screenshots** and logs. The point is - the "a" seems to be holding for both screenshot and logs, making it sound weird.

Comment (vaibhavsingh97) [reply docs/ISSUE_TEMPLATE.md]: Sure!! 😆

**Comment by vaibhavsingh97** [comment]
@nikhita Updated 😄

**Review by nikhita — APPROVED** [review]
LGTM

**Pull request merged**
